### PR TITLE
Update required package list for fedora-minimal

### PR DESCRIPTION
**Pull request by jpouellet**
Tested with separate templates for sys-net and sys-firewall based on fedora-28-minimal.

### CODE REVIEW

File: managing-os/templates/fedora-minimal.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ As expected, the required packages are to be installed in the running template w
 Use case | Description | Required steps
 --- | --- | ---
 **Standard utilities** | If you need the commonly used utilities | Install the following packages: `pciutils` `vim-minimal` `less` `psmisc` `gnome-keyring`
-**FirewallVM** | You can use the minimal template as a [FirewallVM](/doc/firewall/), such as the basis template for `sys-firewall` | No extra packages are needed for the template to work as a firewall.
-**NetVM** | You can use this template as the basis for a NetVM such as `sys-net` | Install the following packages:  `NetworkManager-wifi` `wireless-tools` `dejavu-sans-fonts` `notification-daemon`.
+**FirewallVM** | You can use the minimal template as a [FirewallVM](/doc/firewall/), such as the basis template for `sys-firewall` | Install at least `qubes-core-agent-networking`, and also `qubes-core-agent-dom0-updates` if you want to use it as the updatevm (which is normally sys-firewall).
```
Comment (marmarek) [inline managing-os/templates/fedora-minimal.md]: Those packages exists in R4.0 only (in 3.2 it was all part of qubes-core-agent/qubes-core-vm). Do we want this table duplicated in version-specific section?

cc @andrewdavidwong

Comment (andrewdavidwong) [reply managing-os/templates/fedora-minimal.md]: > Those packages exists in R4.0 only (in 3.2 it was all part of qubes-core-agent/qubes-core-vm). Do we want this table duplicated in version-specific section?

Yes.

@jpouellet, please revise in accordance with:

https://www.qubes-os.org/doc/doc-guidelines/#version-specific-documentation

**Comment by jpouellet** [comment]
Like that?

**Comment by andrewdavidwong** [comment]
Much better. I would put the 4.0 table under the 4.0 section, but I won't let the perfect be the enemy of the good. ;)

**Comment by andrewdavidwong** [comment]
Formatting looks good. Any remarks on content, @marmarek?

**Comment by jpouellet** [comment]
Updated. (I think that's what you meant...)

**Pull request merged**